tests/unit_tests_old/tests_launch/test_launch.py

```diff
@@ -1542,3 +1542,41 @@ def test_launch_git_version_default_main(
     )
 
     assert "main" in str(mock_with_run_info.args[0].git_version)
+
+
+def test_noop_builder(
+    live_mock_server,
+    test_settings,
+    mocked_fetchable_git_repo_main,
+    runner,
+    monkeypatch,
+):
+    launch_config = {"build": {"type": "noop"}, "registry": {"url": "test"}}
+    api = wandb.sdk.internal.internal_api.Api(
+        default_settings=test_settings, load_settings=False
+    )
+    monkeypatch.setattr(
+        wandb.sdk.launch.launch,
+        "LAUNCH_CONFIG_FILE",
+        "./config/wandb/launch-config.yaml",
+    )
+
+    with runner.isolated_filesystem():
+        os.makedirs(os.path.expanduser("./config/wandb"))
```

Comment:
expanduser isn't doing anything here or in the following line, is it?

Reply:
Ah nice catch, nor is it in the line below. Fixed

Reply:
Ive also removed them from the test I grabbed them from.